bin/sandbox

```diff
@@ -93,16 +100,16 @@ unbundled bundle install --gemfile Gemfile
 
 unbundled bundle exec rake db:drop db:create
 
-unbundled bundle exec rails generate spree:install \
+unbundled bundle exec rails generate solidus:install \
   --auto-accept \
   --user_class=Spree::User \
   --enforce_available_locales=true \
   --with-authentication=false \
   --payment-method=none \
   $@
 
-unbundled bundle exec rails generate solidus:auth:install --auto-accept
-unbundled bundle exec rails generate ${extension_name}:install --auto-accept
+unbundled bundle exec rails generate solidus:auth:install --auto-run-migrations
+unbundled bundle exec rails g solidus_starter_frontend:install
```

Comment:
An alternative to running the `solidus_starter_frontend:install` install command is to add the assets directly to `app/views/spree/layouts/spree_application.html.erb` and then strip them from the layout when running the generator. Let me know if you prefer this alternative.

Reply:
Not sure to get what you mean here, do you have an example?

Reply:
@kennyadsl I was easier to create a separate PR implementing that change :) Please see https://github.com/nebulab/solidus_starter_frontend/pull/170. Thanks!

Reply:
I'd just go with this PR's route for now.